doc/source/cluster/kubernetes/examples/mnist-training-example.md

```diff
@@ -0,0 +1,122 @@
+(kuberay-mnist-training-example)=
+
+# Train MNIST on a Neural Network with CPUs on Kubernetes
+
+This guide runs a sample Ray Train workload with CPUs on Kubernetes infrastructure.
+
+## Step 1: Install KubeRay operator
+
+Follow [this document](kuberay-operator-deploy) to install the latest stable KubeRay operator from the Helm repository. After installation, you should see the operator running:
+
+```sh
+# Confirm that the operator is running in the namespace `default`.
+kubectl get pods
+# NAME                                                      READY   STATUS              RESTARTS   AGE
+# kuberay-operator-6dddd689fb-ksmcs                         1/1     Running             0          48s
+```
+
+## Step 2: Create a RayJob
+
+A RayJob consists of a RayCluster custom resource and a job that can be submitted to the RayCluster. With RayJob, KubeRay creates a RayCluster and submit a job when the cluster is ready. Here, we provide a CPU-only RayJob description yaml file for the MNIST training on a Neural Network.
+
+```sh
+# Download `ray-job.pytorch-mnist.yaml`
+curl -LO https://raw.githubusercontent.com/ray-project/kuberay/v1.0.0/ray-operator/config/samples/pytorch-mnist/ray-job.pytorch-mnist.yaml
```

Comment:
https://raw.githubusercontent.com/ray-project/kuberay/v1.0.0/ray-operator/config/samples/pytorch-mnist/ray-job.pytorch-mnist.yaml is 404 not found.

Reply:
Solved